patches/ui-base-accelerators-accelerator.cc.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/ui/base/accelerators/accelerator.cc b/ui/base/accelerators/accelerator.cc
-index c33829d85745e595e3da8d9913f6e75b05e6c4a9..8b081685880f3a947c463bcc90196dcee9ba98ba 100644
+index c33829d85745e..8c5c93f773597 100644
 --- a/ui/base/accelerators/accelerator.cc
 +++ b/ui/base/accelerators/accelerator.cc
-@@ -388,6 +388,7 @@ std::u16string Accelerator::ApplyLongFormModifiers(
-   // Note that we use 'else-if' in order to avoid using Ctrl+Alt as a shortcut.
-   // See https://devblogs.microsoft.com/oldnewthing/20040329-00/?p=40003 for
-   // more information.
+@@ -380,6 +380,7 @@ std::u16string Accelerator::KeyCodeToName() const {
+ 
+ std::u16string Accelerator::ApplyLongFormModifiers(
+     const std::u16string& shortcut) const {
 +  BRAVE_UI_BASE_ACCELERATOR_APPLY_LONG_FORM_MODIFIERS_
```

Comment:
Moved this to the start of the function, now we don't reuse any of the chromium code.